source/agora/common/EnrollmentManager.d

```diff
@@ -318,24 +335,45 @@ unittest
     );
     Hash secondHash = hashFull(secondTx);
 
+    // create the third transaction
+    Transaction thirdTx = Transaction(
+        TxType.Freeze,
+        [Input(Hash.init, 0)],
+        [Output(Amount(90_000), key_pairs[1].address)]
```

Comment:
`[Output(Amount.FreezeAmount, key_pairs[1].address)]`
The freeze amount must be a minimum of 40,000 BOA.